### PR TITLE
bugtool: Add counters to iptables-save output

### PR DESCRIPTION
**Pull request by joestringer**
These can be really handy for debugging some classes of problems where
iptables rules that we expect to be hit are in fact not hit.

---
This change is [Reviewable](https://reviewable.io/reviews/cilium/cilium/8747)

### CODE REVIEW

**Review by ianvernon — APPROVED** [review]

**Comment by joestringer** [comment]
test-me-please

File: bugtool/cmd/configuration.go [hunk 1]
```diff
@@ -85,7 +85,7 @@ func defaultCommands(confDir string, cmdDir string, k8sPods []string) []string {
 		"journalctl -u cilium*",
 		"journalctl -u kubelet",
 		// iptables
-		"iptables-save",
+		"iptables-save -c",
```
Comment (jrajahalme) [inline bugtool/cmd/configuration.go:88]: IMO we should do the same for ip6tabels-save

**Review by nebril — APPROVED** [review]

**Comment by joestringer** [comment]
test-me-please

**Comment by ianvernon** [comment]
Merging, failure is unrelated.

**Pull request merged**